.pipelines/pipeline.yaml

```diff
@@ -322,24 +322,38 @@ stages:
       displayName: Cilium
       pipelineBuildImage: "$(BUILD_IMAGE)"
       testDropgz: ""
+      clusterType: swift-no-kube-proxy-up
       clusterName: "ciliume2e"
+      vmSize: Standard_B2ms
+      k8sVersion: ""
+      windowsOsSku: ""
+      dependsOn: 'containerize'
 
   - template: singletenancy/cilium-overlay/cilium-overlay-e2e-job-template.yaml
     parameters:
       name: "cilium_overlay_cilium_e2e"
       displayName: Cilium on AKS Overlay
       pipelineBuildImage: "$(BUILD_IMAGE)"
       testDropgz: ""
+      clusterType: overlay-no-kube-proxy-up
       clusterName: "overlaye2e"
+      vmSize: Standard_B2ms
+      k8sVersion: ""
+      windowsOsSku: ""
+      dependsOn: 'containerize'
 
   - template: singletenancy/aks-swift/e2e-job-template.yaml
     parameters:
       name: "aks_swift_e2e"
       displayName: AKS Swift Ubuntu
       pipelineBuildImage: "$(BUILD_IMAGE)"
       testDropgz: ""
+      clusterType: byocni-up
```

Comment:
swift-podsubnet-up

Reply:
We can go ahead and rename the first two to `cilium-podsubnet-up` and `cilium-overlay-up`. I think we could keep this one as `byocni-up`, because `swift-podsubnet-up` might be misleading. We have an existing target called `swift-up` which brings up a swift cluster. 

If you don't want to keep `byocni-up` -- Maybe we could go with `swift-byocni-up`, to clearly indicate that it's byocni but just need to make sure it matches in all other places.

Reply:
`swift-byocni-up` makes sense